docs/UserGuide.md

```diff
@@ -556,6 +580,50 @@ You can find your sleep record on a specific date in AtheltiCLI.
 
 * `find-sleep 2021-09-01`
 
+### 🎯 Setting Sleep Goals:
+
+`set-sleep-goal`
+
+You can set goals for your sleep AthletiCLI by setting the target duration specified in minutes. Tracking can be done for the past day, week, month or year.
+
+**NOTE: Only one sleep goal can be set for each time period.**
+
+**Syntax:**
+* `set-sleep-goal type/TYPE period/PERIOD target/TARGET`
+
+**Parameters:**
+* TYPE: The type of sleep goal. It must be the following: `duration`.
+* PERIOD: The period for which you want to set a goal. It must be one of the following: `daily, weekly, monthly, yearly`. Only sleeps that are recorded within the period from the current timewill be counted towards the goal.
+* TARGET: The target value. It must be a positive number. For duration, it is in minutes.
+
+**Examples:**
+* `set-sleep-goal type/duration period/daily target/420` Sets a goal of sleeping 7 hours per day.
+* `set-sleep-goal type/duration period/weekly target/2940` Sets a goal of sleeping 49 hours per week. 
```

Comment:
hh:mm format would be more user-friendly here. This way, the user has to calculate the minutes each time. Probably out of scope for the final submission.

Reply:
Good suggestion, but would require some reworking and testing so wont add that feature for the submission.